### ISSUE TITLE
Unable to create or use codenvy

### ISSUE THREAD

**Issue by MifPT** [codenvy/codenvy#2599]
Hi,
I've just created an account and I'm new at this and was about to create a stack when it seems the site crashed.
After a couple minutes I tried the site again and I unable to login with my user name "MFerreira" Authentication failed...
I thought : "ok, maybe I typed the wrong password, no problem I'll reset it"
Try the reset and it tells me the email "is not registered in the system".
???
I'm unable to create another account, also, because "the email already exists."
So, basically I'm stuck on a loop.

Please see attached PDF with print screens.

Any help would be appreciated.

Thank you,
Miguel Ferreira

RESOLVED:  I needed to type the email with small caps.

**Comment by MifPT** [comment]
RESOLVED: I needed to type the email with small caps.
